components/list/demo/list-demo-drag-n-drop-usage.js

```diff
@@ -70,6 +71,7 @@ class ListDemoDragNDropUsage extends LitElement {
 				${repeat(this.list, (item) => item.key, (item) => html`
 					<d2l-list-item
 						key="${ifDefined(item.key)}"
+						drop-destination="${ifDefined(this._dropLocation || item.key)}"
```

Comment:
Passing this information in the demo doesn't appear to work properly when I test with VoiceOver on Chrome.